lib/Fmt_ast.ml

```diff
@@ -1561,6 +1561,14 @@ and fmt_expression c ?(box = true) ?pro ?epi ?eol ?parens ?(indent_wrap = 0)
   let {pexp_desc; pexp_loc; pexp_attributes; _} = exp in
   update_config_maybe_disabled c pexp_loc pexp_attributes
   @@ fun c ->
+  ( match pexp_desc with
```

Comment:
If the numbers of free variables is reasonable I think that this block can be extracted to a function

Reply:
done, I've moved that code in Cmts.ml and didn't export the other function